packages/core/contracts/optimistic-asserter/implementation/escalation-manager/BaseEscalationManager.sol

```diff
@@ -33,7 +33,7 @@ contract BaseEscalationManager is EscalationManagerInterface {
         emit PriceRequestAdded(identifier, time, ancillaryData);
     }
 
-    function assertionResolved(bytes32 assertionId, bool assertedTruthfully) public {}
+    function assertionResolved(bytes32 assertionId, bool assertedTruthfully) public virtual override {}
 
-    function assertionDisputed(bytes32 assertionId) public {}
+    function assertionDisputed(bytes32 assertionId) public virtual override {}
```

Comment:
Nice.
Should we make every function of every EM implementation virtual?

Reply:
So far we are keeping flat inheritance structure for simplicity, thus virtual is needed only in Base EM. But should reconsider if expecting external EM implementations inheriting some of EM's from this repo

Reply:
I think this makes sence!